Pinta.Core/Actions/EditActions.cs

```diff
@@ -263,6 +263,7 @@ private void HandlerPintaCoreActionsEditCopyActivated (object sender, EventArgs
 					return;
 			
 				ImageSurface dest = new ImageSurface (Format.Argb32, rect.Width, rect.Height);
+				Pinta.Core.Utilities.CheckHandleImageSurface (dest);
```

Comment:
It might be good to also wrap up the construction of the `ImageSurface` too, to reduce the chance of this check being missed as code shifts around. 
`CairoExtensions` is also probably a good place for this to go,
e.g. `ImageSurface dest = CairoExtensions.CreateImageSurface(Format.Argb32, rect.Width, rect.Height);`

Reply:
Def thought about that. I didn't want to change things up too much.  But I'll do that

Reply:
done